web/pages/Success.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react'
+import { useSearchParams } from 'react-router-dom'
+import DisplayOrderProducts from '../components/DisplayOrderProducts.jsx'
+import cartService from '../services/cart.service.js'
+import productService from '../services/product.service.js'
+
+export default function Success() {
+  const [searchParams] = useSearchParams()
+  const [orderProducts, setOrderProducts] = useState([])
+  const orderId = searchParams.get('orderID')
+
+  useEffect(() => {
+    const getOrderInfo = async () => {
```

Comment:
you could add a line here to do nothing if orderId is null
```suggestion
    if (!orderId) {
      return;
    }
    const getOrderInfo = async () => {
```